internal/lox/interpreter.go

```
@@ -306,6 +306,49 @@ func Eval(node Node, environment *Environment, locals Locals, stdout io.Writer,
+
+		methods := make(map[string]*UserFunction)
+		for _, method := range n.Methods {
+			function := NewUserFunction(method, environment, locals)
+			methods[method.Name.Lexeme] = function
+			if method.Name.Lexeme == "init" {
+				function.IsInitializer = true
+			}
+		}
+
+		userClass := &UserClass{Name: n.Name.Lexeme, Methods: methods}
+		environment.Assign(n.Name, userClass)
```

golangci-lint (1.62.2)

Error return value of `environment.Assign` is not checked

(errcheck)